scripts/gen-deps-list.js

```diff
@@ -38,33 +38,35 @@ async function main() {
     throw new Error(`Could not find changelog block in ${changelogConfig.path}`)
   }
 
-  const lines = block.split(/\s*\r?\n\s*/).map(line => {
-    const items = trim(line, '- ').split(/\s+/)
+  block.split(/\r?\n/).forEach(rawLine => {
```

Comment:
I believe you can also remove `\r`, it should be properly handled by `String#trim()`.